pages/tutorials/getting_started.md

```diff
@@ -82,11 +82,10 @@ To create a pipeline:
 
     <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/powershell-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add PowerShell Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-1. Accept the pre-filled defaults by selecting *Create Pipeline*. This associates the example repository with your new pipeline.
-1. Accept the pre-filled defaults by selecting _Save and Build_. This adds a step to upload the full pipeline definition from the repository.
-1. In the modal that opens, create a build using the pre-filled details.
+1. On the _New Pipeline_ page, leave all fields with their pre-filled default values and select _Create Pipeline_. This associates the example repository with your new pipeline, and adds a step to upload the full pipeline definition from the repository.
+1. On the next page showing your pipeline name, click _New Build_ and in the modal that opens, create a build using the pre-filled details.
```

Comment:
I think there are too many actions in this sentence. I think you could probably just split it in two:

```suggestion
1. On the next page showing your pipeline name, click _New Build_. In the modal that opens, create a build using the pre-filled details.
```

I generally try to keep each bullet to one action, so you might also consider splitting it into a new step. I don't think that's necessary here, though.

What do you think?

Reply:
Absolutely, I'm all in for improved readability.
I guess I'm still at the stage of gauging the existing style of writing throughout the docs since it's a little varied.